seed.js

```diff
@@ -1,12 +1,12 @@
 var data = require('./seed_data.js');
 var mongoose = require('mongoose');
-var Stories = require('./db/models/story.js');
+var Songs = require('./db/models/song.js');
 
-mongoose.connect('mongodb://localhost/hackednews');
```

Comment:
I see the mongoose.connect here, so maybe disregard the comment about mongoose.connect above. I'm still trying to get better at separation of concerns, so I am not an expert by any means, but it may be better to keep the connection to mongoose in the schema file.